### PR TITLE
Introduced post processing

### PR DESCRIPTION
**Pull request by prasankh**
[ODSC-61573](https://jira.oci.oraclecorp.com/browse/ODSC-61573)

Following schema can be used to set min value for the forecast value
```
postprocessing:
 enabled: true
 steps:
   set_min_forecast: 0
```

### CODE REVIEW

File: ads/opctl/operator/lowcode/forecast/model/prophet.py [hunk 1]
```diff
@@ -81,6 +81,13 @@ def _train_model(self, i, series_id, df, model_kwargs):
             )
 
             data = self.preprocess(df, series_id)
+            cap = data['y'].max() * 1.2
```
Comment (ahosler) [inline ads/opctl/operator/lowcode/forecast/model/prophet.py:84]: Do we need to have a cap?

File: ads/opctl/operator/lowcode/forecast/model/prophet.py [hunk 2]
```diff
@@ -81,6 +81,13 @@ def _train_model(self, i, series_id, df, model_kwargs):
...
+            if self.spec.postprocessing.enabled:
+                floor = self.spec.postprocessing.steps.set_min_forecast
+            else:
+                floor = data['y'].min() * 0.8
```
Comment (ahosler) [inline ads/opctl/operator/lowcode/forecast/model/prophet.py:88]: Shouldn't we leave this to the framework? Why 0,8?

File: ads/opctl/operator/lowcode/forecast/schema.yaml [hunk 3]
```diff
@@ -317,6 +317,25 @@ spec:
               required: false
               default: false
 
+    postprocessing:
```
Comment (ahosler) [inline ads/opctl/operator/lowcode/forecast/schema.yaml:320]: This is a great way to add it 👍

File: ads/opctl/operator/lowcode/forecast/model/prophet.py (hunk 1 above)
Comment (prasankh) [reply ads/opctl/operator/lowcode/forecast/model/prophet.py]: Yes, It's mandatory with logistic growth function. Removed logistic growth function for now as it fails during generating local explanations.

File: ads/opctl/operator/lowcode/forecast/model/prophet.py (hunk 2 above)
Comment (prasankh) [reply ads/opctl/operator/lowcode/forecast/model/prophet.py]: It's mandatory with logistic growth function. Removed logistic growth function for now.

**Review by codeloop — APPROVED** [review]

**Pull request closed without merging**